### PR TITLE
reorganized installation documentation and made some revisions to con…

### PR DESCRIPTION
**Pull request by BrendaNoonan**
…tent

### CODE REVIEW

File: presto-docs/src/main/sphinx/getting-started-original.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+===============
```
Comment (BrendaNoonan) [inline presto-docs/src/main/sphinx/getting-started-original.rst]: Please ignore this file. It is a copy of the original. Changes are in getting-started.rst.

File: presto-docs/src/main/sphinx/installation/installation-ambari-original.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
+************************************
+Automated Installation for a Cluster
```
Comment (BrendaNoonan) [inline presto-docs/src/main/sphinx/installation/installation-ambari-original.rst]: Please ignore this file. It's a copy of the original. Changes are in installation-ambari-original.rst

File: presto-docs/src/main/sphinx/installation/installation-yarn-original.rst [hunk 3]
```diff
@@ -1,6 +1,6 @@
-======================================
- Presto on Yarn-based cluster
-======================================
+﻿==============================================
```
Comment (BrendaNoonan) [inline presto-docs/src/main/sphinx/installation/installation-yarn-original.rst]: Please ignore this file. Changes are in installation-yarn-xxxx.rst files.

**Comment by mattsfuller** [comment]
Added to https://github.com/Teradata/presto/pull/244

**Pull request closed without merging**